README.md

```diff
@@ -240,24 +267,21 @@ export default async function Page() {
 
 ### Encrypting data with a lock context
 
-To encrypt data with a lock context, pass the lock context object as a parameter to the `encrypt` function.
+To encrypt data with a lock context, call the optional `withLockContext` method on the `encrypt` function and pass the lock context object as a parameter.
 
 ```typescript
 const ciphertext = await eqlClient.encrypt('plaintext', {
   table: 'users',
   column: 'email',
-  lockContext,
-})
+}).withLockContext(lockContext)
 ```
 
 ### Decrypting data with a lock context
 
-To decrypt data with a lock context, pass the lock context object as a parameter to the `decrypt` function.
+To decrypt data with a lock context, call the optional `withLockContext` method on the `decrypt` function and pass the lock context object as a parameter.
```

Comment:
I suggest putting a big notice at the top of this section that the **same** lock context must be provided to decrypt as was used during encrypt otherwise decryption will fail.